tests/unit/emulatemb.cxx

```diff
@@ -483,5 +748,16 @@ int main(int argc, char** argv)
   testDragAndTimeout();
   testDragAndRelease();
 
+  testMBEMKeyPress(false);
+  testMBEMKeyPress(true);
+
+  testMBEMModifier(false);
+  testMBEMModifier(true);
+
+  testMBEMClick();
+  testMBEMKeyAndClick();
+  testMBEMRightClick();
+  testMBEMMultiClick();
```

Comment:
I'm lacking tests with more keys pressed. I.e. if I press Ctrl+Alt and then left click. Should that be a middle click? Or a left click? What does other systems do in that case?